pages/common/evil-winrm.md

```diff
@@ -0,0 +1,38 @@
+# evil-winrm
+
+> Windows Remote Management (WinRM) shell for pentesting.
+> More information: <https://github.com/Hackplayers/evil-winrm>.
+
+- Connect to a victim host:
+
+`evil-winrm --ip {{ip}} --user {{user}} --password {{password}}`
+
+- Connect to a victim host, passing the password hash:
+
+`evil-winrm --ip {{ip}} --user {{user}} --hash {{nt_hash}}`
+
+- Connect to a victim host, specifying folders for scripts and executables:
+
+`evil-winrm --ip {{ip}} --user {{user}} --password {{password}} --scripts {{path/to/scripts}} --executables {{path/to/executables}}`
+
+- Connect to a victim host, using SSL:
+
+`evil-winrm --ip {{ip}} --user {{user}} --password {{password}} --ssl --pub-key {{path/to/pubkey}} --priv-key {{path/to/privkey}}`
+
+> Once connected, we get a PowerShell prompt on the victim host.
```

Comment:
This is problematic. Why not put it in the top description as well?

Also, I would suggest to change `victim host` to something else in the whole document. Perhaps just `host`.

Reply:
Not sure I understand what you're getting at here @zdroid.